supernovacontroller/sequential/i3c.py

```diff
@@ -1399,9 +1400,10 @@ def ccc_broadcast_setxtime(self, timing_parameter):
             responses = self.controller.sync_submit([
                 lambda id: self.driver.i3cBroadcastSETXTIME(
                     id,
-                    timing_parameter,
                     self.push_pull_clock_freq_mhz,
                     self.open_drain_clock_freq_mhz,
+                    timing_parameter,
```

Comment:
Does the SubCommand Byte for SETXTIME is always interpreted as a timing parameter?